libs/telemetry/src/main/java/org/opensearch/telemetry/tracing/SpanCreationContext.java

```diff
@@ -28,6 +28,11 @@ public final class SpanCreationContext {
      */
     private SpanCreationContext() {}
 
+    SpanCreationContext(String spanName, Attributes attributes) {
```

Comment:
Please remove that, we've `server()` and `client()` factory methods, please use those